src/lib/channel.ts

```diff
@@ -34,6 +34,15 @@ export async function init() {
       } catch (err) {
         console.error(`Failed processing invite codes message: ${err}`, message);
       }
+    } else if (message.event === "calendar_selected") {
+      // @TODO: set selected calendar.
```

Comment:
That's cool, then the frontend can change it's UI as soon as the backend is ready to throw all the events at it